src/apps/beefy/positions.ts

```diff
@@ -74,7 +74,7 @@ const beefyAppTokenDefinition = ({
     vault.tokenAddress?.toLowerCase() ??
     networkIdToNativeAssetAddress[networkId]
   const tvl = tvls[vault.id]
-  const apy = apys[vault.id]
+  const apy = apyBreakdown[vault.id].totalApy
```

Comment:
I see five or so `*-earnings` vaults that do not have earnings defined do these need a fallback?

Reply:
good idea! I can be a bit more defensive on a few things here